docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -850,14 +862,41 @@ Default: `false`
 
 ##### --file_expiration_value_ttl_overrides_table_ttl
 
-When set to true, allows files to expire purely based on their value-level TTL expiration time (even if it is lower than the table TTL). This is useful for times where a file needs to expire earlier than its table-level TTL would allow. If no value-level TTL metadata is available, then table-level TTL will still be used. 
+When set to true, files expire based on their value-level TTL expiration time, even if it is lower than the table TTL. This is useful when a file needs to expire earlier than its table-level TTL would allow. If no value-level TTL metadata is available, then table-level TTL is still used.
 
 {{< warning title="Warning">}}
 Use of this flag can potentially result in expiration of live data - use at your discretion.
 {{< /warning >}}
 
 Default: `false`
 
+## Using postgresql.conf
+
+Like PostgreSQL, YugabyteDB uses the postgresql.conf settings file to apply server configuration settings to new server instances.
+
+For information on using postgresql.conf to configure servers, refer to [Server Configuration](https://www.postgresql.org/docs/11/runtime-config.html) in the PostgreSQL documentation.
+
+The server settings options for YugabyteDB are the same as for PostgreSQL, with the following exceptions.
+
+### log_line_prefix
+
+YugabyteDB supports the following additional options for the log_line_prefix setting:
+
+- %C = cloud name
+- %R = region / data center name
+- %Z = availability zone / rack name
+- %U = cluster UUID
+- %N = node and cluster name
+- %H = current hostname
+
+For information on using log_line_prefix, refer to [log_line_prefix](https://www.postgresql.org/docs/11/runtime-config-logging.html#GUC-LOG-LINE-PREFIX) in the PostgreSQL documentation.
+
+### suppress_nonpg_logs (boolean)
+
+When set, suppresses logging of non-PostgreSQL output.
```

Comment:
I think we should mention that it suppress logging of non-PostgresSQL output to the Postgres log file. Current wording might make it sound like it prevents all non-PostgresSQL logs to all files.